### DISCUSSION TITLE
B917795C-3190-BC23-51AC-C50A355A2105

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#139] [General]
# B917795C-3190-BC23-51AC-C50A355A2105

This addon compares the orbital accuracy of Celestia's different versions. Note, there some oddities caused by multiple objects being in the same place, mostly it involves the textures. These were purposely left in. This addon was updated from its original 2019 version to include textures files that are default to Celestia version 1.6.1 due to those textures being updated. All of the data and textures are from the original Celestia versions 1.0 and 1.2.4-1.6.1 and the only edit to the files were to make version 1.0's definitions work (due to the original SSC file having a different structure) and have each object have the correct texture. Also version 1.0's Saturn will appear as a solid black sphere if ring shadows are on due to the ring texture being a JPEG rather then a PNG image.

https://celestia.mobi/addon?item=B917795C-3190-BC23-51AC-C50A355A2105

**Comment by Adiputr** [comment]
I want to explore beyond ang
